components/MapView.js

```diff
@@ -77,6 +77,13 @@ var MapView = React.createClass({
     rotateEnabled: PropTypes.bool,
 
     /**
+     * If `true` the map will be cached to an Image for performance
+     * Default value is `false`.
+     *
+     */
+    cacheEnabled: PropTypes.bool,
```

Comment:
Can you add that this is android only?

On that topic... is there anything analogous to this on mapkit that you know of?